### PR TITLE
Modify 500001_axialracing_ax10 to utilize rc.ugv_defaults.

### PR DESCRIPTION
**Pull request by mcsauder**
Hi,

This PR modifies the 500001_axialracing_ax10 config file to utilize rc.ugv_defaults. This modification allows deprecating both rc.axialracing_ax10_apps and rc.axialracing_ax10_defaults to save a few hundred bytes of flash space.

To attend to the work taking place in the deprecated files, the `ekf2 start` previously occurring in rc.axialracing_ax10_apps now gets called within rc.ugv_apps, and `PWM_RATE 50` and  `PWM_OUT 1234` which were being set in rc.axialracing_ax10_defaults are being set in rc.ugv_defaults.

Please let me know if you have any comments/questions on this PR.

-Mark

### CODE REVIEW

**Comment by mcsauder** [comment]
@dagar

**Comment by LorenzMeier** [comment]
Awesome, thanks!

**Comment by mcsauder** [comment]
I discovered an oversight and corrected it by updating `ROMFS/tap_common/init.d/rcs` to utilize `rc.ugv_apps` rather than `rc.axialracing_ax10_apps`.  I also updated the logic to check for `ugv` vehicle type:
  `if [ $VEHICLE_TYPE == rover -o $VEHICLE_TYPE == ugv]` 

It appears at a glance that this could be shortened to only check `if [ $VEHICLE_TYPE == ugv ]`
Let me know if it is preferable to change that logic check.

Thanks.

File: ROMFS/tap_common/init.d/rcS [hunk 1]
```diff
@@ -313,9 +313,9 @@ then
 fi
 
 #
-# Rover setup
+# UGV/Rover setup
 #
-if [ $VEHICLE_TYPE == rover ]
+if [ $VEHICLE_TYPE == rover -o $VEHICLE_TYPE == ugv]
```
Comment (dagar) [inline ROMFS/tap_common/init.d/rcS]: Do we need both VEHICLE_TYPEs?

Comment (mcsauder) [reply ROMFS/tap_common/init.d/rcS]: I'm glad you think that as well. I'll pull out the check for rover!

Comment (mcsauder) [reply ROMFS/tap_common/init.d/rcS]: As a note, when I implemented that logic update I searched the repo to see if anywhere in the code base `VEHICLE_TYPE` is set to `rover`, and there are no occurrences, so this was likely dead code previously anyway.

**Pull request merged**